packages/samples/headless-commerce-ssr-remix/projects.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
isnt this supposed to be "project.json" ? and not "project**s**.json". Not sure if it matters

Reply:
I was also not running the correct build command... that would explain why the branch isn't building on the CI 😂